src/templates/ar/2019/base.html

```diff
@@ -97,10 +146,45 @@
         var menuIcon = document.querySelector('.menu-icon');
         var menuIconInitialSrc = document.querySelector('.menu-icon').src;
         var menuIconSrcClose = document.querySelector('.menu-icon').dataset.srcClose;
+        var menuBtn = document.querySelector('.menu-btn');
+        var menuNav = document.querySelector('#Menu');
+        var menu = document.querySelector('#Menu');
+
+        function triggerMenu() {
+          menuBtn.click();
+        }
 
-        document.querySelector('.menu-btn').addEventListener('click', function() {
+        menuBtn.addEventListener('click', function() {
           var menuOpen = document.body.classList.toggle('menu-open');
           menuIcon.src = menuOpen ? menuIconSrcClose : menuIconInitialSrc;
+          menuBtn.setAttribute('aria-expanded',menuOpen);
+          var ariaLabel = menuOpen ?  "Close the menu" : "Open the menu";
```

Comment:
I think we should avoid setting text in JS if possible. Is there a way to have open and close messages as data attributes on the element instead? I think it'd be easier for translators to only have to worry about translating markup and not have to comb through JS looking for strings.

Having a language-agnostic script would also enable us to pull this block into a common template and avoid duplicating the code for every language. Could you also do that in this PR?

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/666/commits/1d5172131674bfb0a4e5f21ed6431d4a09e56eef

Reply:
The script still appears to be duplicated for every language. Could we try to reduce those redundancies?

Reply:
I think that would be a good idea. However I think that’s separate to this PR to be honest because we should generalise all scripts (and potentially other common things).

Appreciate this extra JS may be the straw that’s broken the camel’s back but still feel it should be a separate PR to restructure the templates and build process. Happy to raise an issue for that so we don’t forget?

Reply:
Ok SGTM

Reply:
Raised #673 for that work.